src/renderer/styles/responsive.css

```
@@ -98,7 +98,16 @@
 		gap: 8px;
 	}
 	.form-actions {
-		margin: 0 -16px;
-		padding: 16px;
+		position: fixed;
+		left: 16px;
+		right: 16px;
+		bottom: calc(
+			env(safe-area-inset-bottom) +
```

Stylelint (17.12.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)